### PR TITLE
Anson_adding_template_interaction

### PR DESCRIPTION
**Pull request by ansonliang55**
@jyotheeshwar please review

### CODE REVIEW

File: template.feature [hunk 1]
```diff
@@ -1 +1,163 @@
...
+Textbox:
+Enter text in textbox
+I enter "<string>" in textbox <context> "<value>"
+	Examples:
+	| context    |
+	| in section |
+	| with id    |
+	eg: I enter "ehealthuser1" in textbox with id "profile-form-username"
+
+Delete text
```
Comment (jyotheeshwar) [inline template.feature]: Delete can be replaced by just entering "" in the enter text scenario. can be removed.

File: template.feature [hunk 2]
```diff
@@ -1 +1,163 @@
...
+	Examples:
+	| position  |
+	| preceding |
+	| following |
+eg. I should see "hello" preceding "how are you"
+
+
+Textbox:
+Enter text in textbox
+I enter "<string>" in textbox <context> "<value>"
```
Comment (jyotheeshwar) [inline template.feature]: add a field for textbox name. 

`I enter "<string>" in "<textbox_name>" <context> "<value>"`

File: template.feature [hunk 3]
```diff
@@ -1 +1,163 @@
...
+	| with id    |
+	eg: I enter "ehealthuser1" in textbox with id "profile-form-username"
+
+Delete text
+I delete "<string>" in textbox <context> "value"
+	eg: I delete "ehealthuser1" in textbox with id "profile-form-username"
+
+# Check for style(error bars)
+
+Check for disabled/enabled
```
Comment (jyotheeshwar) [inline template.feature:78]: maybe add scenario to see if textbox exists/doesn't exist

File: template.feature [hunk 4]
```diff
@@ -1 +1,163 @@
...
+Dropdown:
+Select y from x dropdown
+I select "<string>" from dropdown <context> "<value>"
+	Examples:
+	| context    |
+	| in section |
+	| with id    |
+	eg. I select "1990" from dropdown with id "profile-form-birth-year"
+
+Navigation:
```
Comment (jyotheeshwar) [inline template.feature:93]: I should be `on` page? Not sure.

**Comment by jyotheeshwar** [comment]
+1 otherwise

File: template.feature (hunk 2 above)
Comment (ansonliang55) [inline template.feature]: i feel the textbox_name is redundant when you have the context and value. Maybe provide an example?

**Comment by ansonliang55** [comment]
merged

**Pull request closed without merging**